### ISSUE TITLE
Redesign app settings configuration and categorize it

### ISSUE THREAD

**Issue by juankaromo** [wazuh/wazuh-kibana-app#2385]
Hi team,

Currently the Wazuh app settings are shown in a table that is not visually impressive. The idea of this issue is to redesign this section to be similar to the Kibana configuration section. 

In it, several configuration blocks are categorized, which help us to organize the configurations depending on their type. Also included are a search engine and a category selector to speed up navigation to a particular setting.

Also, with the introduction of the new configuration for Cron Tasks (https://github.com/wazuh/wazuh-kibana-app/issues/2064) we will need a specific block for them.

![image](https://user-images.githubusercontent.com/21195730/87538180-1653a600-c69c-11ea-8a4c-70109d6a2cb4.png)

As a disadvantage that differentiates us from the Kibana configuration, is that we can only apply the change of a setting at a time, so when changing the value of a setting, a save button will appear right next to the control of that parameter. 

In short, we must perform the following steps:

- [x] Switch from table mode to settings list mode
- [x] Adapt the functionality of the Save button when changing a setting
- [ ] Divide settings into category blocks
- [ ] Add the controls and navigation to search and select a category

Regards,

**Comment by jsanchez91** [comment]
Working on branch: [dev/2385_redesing_configuration_view](https://github.com/wazuh/wazuh-kibana-app/tree/dev/2385_redesing_configuration_view)

**Comment by jsanchez91** [comment]
**Update:**

- He created the components of the strangulation
- You created the header component
- He created the category component
- Creating the Setting Information Component
![image (56)](https://user-images.githubusercontent.com/3064506/87800681-fca69000-c84e-11ea-89fd-800744a036a5.png)
